diff --git a/src/home.c b/src/home.c
--- a/src/home.c
+++ b/src/home.c
@@ -482,10 +482,8 @@ on_drag_state_changed (PhoshHome *self)
   case PHOSH_DRAG_SURFACE_STATE_UNFOLDED:
     state = PHOSH_HOME_STATE_UNFOLDED;
     kbd_interactivity = TRUE;
-    if (self->focus_app_search) {
-      phosh_overview_focus_app_search (PHOSH_OVERVIEW (self->overview));
-      self->focus_app_search = FALSE;
-    }
+    phosh_overview_focus_app_search (PHOSH_OVERVIEW (self->overview));
+    self->focus_app_search = FALSE;
     phosh_home_set_background_alpha (self, 1.0);
     break;
   case PHOSH_DRAG_SURFACE_STATE_FOLDED:
